src/react-apps/applications/shared/src/utils/returnInstanceAttachments.ts

```diff
@@ -0,0 +1,37 @@
+import { IAttachment } from './../types/index';
+
+interface IData {
```

Comment:
Any reason these interfaces are declared here and not exported from our interfaces in shared?

Reply:
No good reason. Fixed :)